spec/features/brexit_checker/accounts_spec.rb

```diff
@@ -8,40 +8,6 @@
     stub_request(:get, Services.accounts_api).to_return(status: 200)
   end
 
-  context "without accounts enabled" do
-    let(:mock_results) { %w[nationality-eu] }
-
-    context "/transition-check/saved-results" do
```

Comment:
I figured these might work better as request specs now.
We can drop the mock results and just focus on what happens if you send an HTTP request and the expected response?